health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FileComponent.js

```
@@ -15,6 +15,18 @@ const FileComponent = ({ title, fileName, status, auditDetails, editHandler, del
     const [fileForPreview, setFileForPreview] = useState([]);
     const XlsPreview = Digit.ComponentRegistryService.getComponent("XlsPreview");
 
+
+    function formatDate(dateString) {
+        // Parse the input date string
+        const [day, month, year, time, period] = dateString.split(' ');
+
+        // Remove the ordinal suffix (e.g., "5th" to "5")
+        const dayNumber = parseInt(day, 10);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)